### ISSUE TITLE
Can't install gem nokogiri on Ubuntu 14.04 because of missing headers

### ISSUE THREAD

**Issue by nicwaller** [chef-cookbooks/xml#19]
This cookbook didn't work for me with Ubuntu 14.04. The gem 'nokogiri' fails to install, because the library 'libxml2' fails to compile, because the headers for 'liblzma-dev' and 'zlib1g-dev' are unavailable.

**Steps to reproduce on vanilla Ubuntu 14.04 (on AWS)**

    sudo /opt/chef/embedded/bin/gem install nokogiri -q --no-rdoc --no-ri -v "1.6.5"

**Output**
```
Error executing action `install` on resource 'chef_gem[nokogiri]'
---- Begin output of /opt/chef/embedded/bin/gem install nokogiri -q --no-rdoc --no-ri -v "1.6.5" ----
STDOUT: Building native extensions.  This could take a while...
STDERR: ERROR:  Error installing nokogiri:
    ERROR: Failed to build gem native extension.
Running 'compile' for libxml2 2.9.2... ERROR, review '/opt/chef/embedded/lib/ruby/gems/2.1.0/gems/nokogiri-1.6.5/ext/nokogiri/tmp/x86_64-unknown-linux-gnu/ports/libxml2/2.9.2/compile.log' to see what happened.
```

**Workarounds**
- `sudo apt-get install liblzma-dev zlib1g-dev`
- `sudo /opt/chef/embedded/bin/gem install nokogiri -q --no-rdoc --no-ri -v "1.6.5" -- --use-system-libraries --with-xml2-include=/usr/include/libxml2`

**Suggestion**
Add packages `liblzma-dev`, `zlib1g-dev` to default['xml']['packages'] for debian.

**Comment by dchapman1988** [comment]
:+1:

**Comment by gnchapman** [comment]
I have been searching since last night for a solution to this problem.  All Stack Overflow posts were useless.  Your workaround saved me!  Thank you!

FYI - running clean install of Ubuntu 14.04 on VMWare, with a new installation of Ruby (v1.9.3p484)

**Comment by nicwaller** [comment]
Fixed in bcaf663ca6eeca5b3cee2e90f28851f907abe819
